### PR TITLE
[CI:DOCS] Update doc to explictly mention using ed25519 in ssh keys

### PR DESCRIPTION
**Pull request by wilbeibi**
#### What this PR does / why we need it:

Looks like many people stumble upon this issue https://github.com/containers/podman/issues/8323 (so do I), maybe it worth to mention it in the doc the ssh key pair is ed25519.

#### Which issue(s) this PR fixes:

Fixes [#8323](https://github.com/containers/podman/issues/8323 )

### CODE REVIEW

**Comment by rhatdan** [comment]
/approve
LGTM
Thanks @wilbeibi

File: docs/tutorials/remote_client.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ sudo systemctl enable --now -s sshd
 #### Setting up SSH
 Remote Podman uses SSH to communicate between the client and server. The remote client works considerably smoother using SSH keys. To set up your ssh connection, you need to generate an ssh key pair from your client machine.
 ```
-ssh-keygen
+ssh-keygen -t ed25519  -f $HOME/.ssh/id_rsa
```
Comment (Luap99) [inline docs/tutorials/remote_client.md:63]: The file should not be called is_rsa when it is a ed25519 key. Just use the default name.

Comment (wilbeibi) [reply docs/tutorials/remote_client.md]: Make sense, will update it.

**Comment by flouthoc** [comment]
@wilbeibi Could you please sign the commit.

File: docs/tutorials/remote_client.md [hunk 2]
```diff
@@ -60,9 +60,9 @@ sudo systemctl enable --now -s sshd
 #### Setting up SSH
 Remote Podman uses SSH to communicate between the client and server. The remote client works considerably smoother using SSH keys. To set up your ssh connection, you need to generate an ssh key pair from your client machine.
```
Comment (TomSweeneyRedHat) [inline docs/tutorials/remote_client.md:61]:
```suggestion
Remote Podman uses SSH to communicate between the client and server. The remote client works considerably smoother using SSH keys. To set up your ssh connection, you need to generate an ssh key pair from your client machine.  *NOTE:* in some instances, using a `rsa` key will cause connection issues, be sure to create an `ed25519` key.
```

**Comment by Luap99** [comment]
Can you squash your commits into one please.

**Pull request merged**